### ISSUE TITLE
OSError: dlopen(encrypt.so, 6): image not found

### ISSUE THREAD

**Issue by Jirawong** [PokemonGoF/PokemonGo-Bot#2642]
Bug found in OSX 
after pull https://github.com/PokemonGoF/PokemonGo-Bot/commit/55846108e235fda2eda370b99c9019f7d89e8619

Traceback (most recent call last):
  File "pokecli.py", line 493, in <module>
    main()
  File "pokecli.py", line 127, in main
    report_summary(bot)
  File "pokecli.py", line 136, in report_summary
    metrics.capture_stats()
  File "/Users/Jirawong/Develop/pokemon/new/PokemonGo-Bot/pokemongo_bot/metrics.py", line 76, in capture_stats
    response_dict = request.call()
  File "/Users/Jirawong/Develop/pokemon/new/PokemonGo-Bot/pokemongo_bot/api_wrapper.py", line 106, in call
    result = self._call()
  File "/Users/Jirawong/Develop/pokemon/new/PokemonGo-Bot/pokemongo_bot/api_wrapper.py", line 63, in _call
    return PGoApiRequest.call(self)
  File "/Users/Jirawong/Develop/pokemon/new/PokemonGo-Bot/src/pgoapi/pgoapi/pgoapi.py", line 209, in call
    request.activate_signature(lib_path)
  File "/Users/Jirawong/Develop/pokemon/new/PokemonGo-Bot/src/pgoapi/pgoapi/rpc_api.py", line 82, in activate_signature
    ctypes.cdll.LoadLibrary(lib_path)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ctypes/__init__.py", line 443, in LoadLibrary
    return self._dlltype(name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ctypes/__init__.py", line 365, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: dlopen(encrypt.so, 6): image not found

**Comment by solderzzc** [comment]
you need manually download a encrypt.so from internet....

**Comment by OVO647** [comment]
Same issue

**Comment by Nepoxx** [comment]
@solderzzc Got the file, where do I put it?

**Comment by simonsmh** [comment]
@Nepoxx Where did you find it? Please send me it thx.
simonsmh@gmail.com 
:)

**Comment by LHagood** [comment]
encrypt.so goes in the main PokemonGo-Bot directory

**Comment by ManiacMike** [comment]
download and just put it under PokemonGo-Bot directory solve my problem
https://slack-files.com/T1U6G9ZEW-F1YV50RGQ-cc979942b7

mac os 10.11.4

**Comment by k4n30** [comment]
#2768

**Comment by nivong** [comment]
https://github.com/PokemonGoF/PokemonGo-Bot/issues/2780

**Comment by veil98** [comment]
Help! I am on Mac.  I got the OP's original error message.  I downloaded the .so file from here:

https://slack-files.com/T1U6G9ZEW-F1YV50RGQ-cc979942b7

I renamed the file to encrypt.dll and put it in pgoapi-master baseline directory.  I then ran:

python pokecli.py -a ptc -u blah -p blah -l “blah” 

And I get the error code: 

 'status_code': 2,
    'unknown6': [{   'response_type': 6, 'unknown2': {   'unknown1': 1L}}]}

Anyone else getting this?
